editor/editor_plugin_settings.cpp

```diff
@@ -179,12 +150,41 @@ void EditorPluginSettings::_cell_button_pressed(Object *p_item, int p_column, in
 	if (p_id == BUTTON_PLUGIN_EDIT) {
 		if (p_column == 4) {
 			String dir = item->get_metadata(0);
-			plugin_config_dialog->config("res://addons/" + dir + "/plugin.cfg");
+			plugin_config_dialog->config(dir);
 			plugin_config_dialog->popup_centered();
 		}
 	}
 }
 
+Vector<String> EditorPluginSettings::_get_plugins(const String &p_dir) {
+	DirAccess *da = DirAccess::create(DirAccess::ACCESS_RESOURCES);
```

Comment:
You could use `DirAccessRef`, this way you don't need to `memdelelte(da);`, it's automatically freed when going out of scope.

Reply:
Did not know, thanks!